examples/README.md

```diff
@@ -0,0 +1,3 @@
+# openid-client-helper Examples
+
+## [Express.js example](express/README.md)
```

Comment:
Not sure why we have this file and docs/README.md

Reply:
changed